src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,135 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private List<Entry<K, V>>[] buckets;
+    private int size;
+
+    public MyHashMap() {
+        buckets = new ArrayList[DEFAULT_CAPACITY];
+        size = 0;
```

Comment:
In Java, declaring the initial value of a variable when it's first declared is optional. When you write private int size = 0;, you're explicitly setting the initial value of size to 0. However, in Java, if you don't initialize an instance variable, it gets a default value based on its type. For int, the default initial value is 0 anyway. So, writing private int size = 0; is redundant in this case because size would already be initialized to 0 by default. You could simply write private int size; to achieve the same result.